printf format issue on RTEMS

**Describe the bug**
When compiling the test app under RTEMS, a build failure occurs:
```
/home/joe/code/cfecfs/github/cfe/modules/cfe_testcase/src/es_task_test.c: In function 'TestCreateChild':
/home/joe/code/cfecfs/github/cfe/modules/cfe_testcase/src/es_task_test.c:89:5: error: format '%d' expects argument of type 'int', but argument 7 has type 'uint32' [-Werror=format=]
     UtAssert_True(countCopy == count || countCopy == count + 1, "countCopy (%d) == count (%d)", countCopy, count);
     ^
```

**To Reproduce**
Build system for RTEMS 4.11 with unit tests enabled, and `es_task_test.c` will fail to build.

**Expected behavior**
Build should succeed.

**System observed on:**
RTEMS 4.11

**Reporter Info**
Joseph Hickey, Vantage Systems, Inc.

--- a/modules/cfe_testcase/src/es_task_test.c
+++ b/modules/cfe_testcase/src/es_task_test.c
@@ -86,7 +86,8 @@ void TestCreateChild(void)
 
     OS_TaskDelay(500);
 
-    UtAssert_True(countCopy == count || countCopy == count + 1, "countCopy (%d) == count (%d)", countCopy, count);
+    UtAssert_True(countCopy == count || countCopy == count + 1, "countCopy (%d) == count (%d)", (int)countCopy,
+                  (int)count);
 }
 
 void TestExitChild(void)

--- a/modules/cfe_testcase/src/time_current_test.c
+++ b/modules/cfe_testcase/src/time_current_test.c
@@ -86,7 +86,7 @@ void TestGetTime(void)
     OS_GetLocalTime(&end);
 
     CFE_TIME_Print(timeBuf1, Time);
-    UtPrintf("The current time is (%d) %s", Time.Seconds, timeBuf1);
+    UtPrintf("The current time is (%ld) %s", (long)Time.Seconds, timeBuf1);
 
     difference = OS_TimeSubtract(end, start);
 

